### ISSUE TITLE
Map

### ISSUE THREAD

**Issue by MG10327** [MG10327/Documentation#35]
### Map

A Map holds key-value pairs where the keys can be any datatype.
A map remembers the original insertion order of the keys.

`const fruits = new Map([`
`["apples", 500],`
`["bananas", 300],`
`["oranges", 200],`
`]`

------------------------------------------------------------------------------------------------------------------------------

### .map()
A method you apply to an array that creates a new array populated with the results of calling a provided function on every element in the original array.

Long story short, you apply this to an array to use a function on each one of the items in it.

For example:
`const array1 = [1,4,9,16]`
// Pass a function to map
`const map1 = array1.map((x) => x * 2);`
`console.log(map1);`
// Since the function is to multiply everything in the OG array by 2, this will output the following. [2, 8, 18, 32]
